web/src/components/charts/BarChart.tsx

```diff
@@ -0,0 +1,52 @@
+import { Bar } from 'react-chartjs-2'
+import Theme from '../../theme'
+
+export default function BarChart({
+  labels,
+  data,
+  title
+}: {
+  labels: Array<string>
+  data: Array<number>
+  title: string,
+}) {
+  const chartData = {
+    labels,
+    datasets: [
+      {
+        label: title,
+        data,
+        backgroundColor: [
+          Theme.palette.canaryYellow.faded,
+          Theme.palette.sherwoodGreen.faded,
+          Theme.palette.skyBlue.faded,
+          Theme.palette.palePink.faded,
+          Theme.palette.petalPurple.faded,
+        ],
+        borderColor: [
+          Theme.palette.canaryYellow.main,
+          Theme.palette.sherwoodGreen.main,
+          Theme.palette.skyBlue.main,
+          Theme.palette.palePink.main,
+          Theme.palette.petalPurple.main,
+        ],
+        borderWidth: 1,
+      },
+    ],
+  }
+
+  const options = {
+    scales: {
+      yAxes: [
+        {
+          ticks: {
+            beginAtZero: true,
+          },
+        },
+      ],
+    },
+    responsive: true,
+  }
+
+  return <Bar data={chartData} options={options} />
+}
```

Comment:
I totally disagree on your comment about skipping out on auto-formatting in . Here's an example of where, for some reason, my files do not have an end of new line. Now I need to manually go in and fix it and re-push. 

https://github.com/Left-on-Read/leftonread/commit/67472ca7a60dfab80ea121d117cdde9e380fb661

Or we need husky again for each project https://github.com/Left-on-Read/leftonread/issues/64

Reply:
Sure, I see, will turn it on. Doesn't your VSCode catch this stuff tho?

Reply:
> Whoa this is a lot of changes to include in one PR. Next time let's try to break it down even if it seems ridiculously small. Much easier to review that way, revert if needed, and determine what this commit did (if something breaks and I need to look back through commits to see what broke something)

I knew you were going to say this 😆 .

Will do in the future. It's just that not all of us push straight to qin 😉 , so making a million little PRs and then having to branch off of all of them is annoying/slow/and makes me dependent upon your reviewing. 

Especially for an application that is not even in production yet.

Reply:
Idk why Github's "quote reply" posted that ^ in this thread.


Anyway, _"Doesn't your VSCode catch this stuff tho?"_ depends if user has X rule in his vscode settings.json. I don't think we can assume that people's editors will have any sort of auto formatting set up.

Reply:
LOL yeah I mean just use your judgement here... but I will genuinely say that eslint completely broke when I booted things up today and I wasn't sure why it all of a sudden broke after not touching things for 2 weeks so when poking through the commits to see if anything changed, it's a huge hassle if it's not clear what a commit is doing because I then actually have to poke through all the files it touches. You also don't have to wait for my review tbh - I'll probably only review stuff that you explicitly request. Also yeah it's super slow having to make a PR for each thing you do, but more commits!

Reply:
Nice work on adding eslint-config. Brilliant idea. It was broken after we hoisted it. I imagine you were seeing same thing as: https://github.com/Left-on-Read/leftonread/issues/58#issuecomment-773818388 

Anyway, @Teddarific understood. We are on the same page. You can trusttttt meeee 😛